_inc/client/at-a-glance/stats.jsx

```diff
@@ -150,7 +162,7 @@ const DashStatsBottom = React.createClass( {
 				<div className="jp-at-a-glance__stats-summary-bestday">
 					<p className="jp-at-a-glance__stat-details">Best overall day</p>
```

Comment:
Unless you're handling that in a different full-sweep -- noticed there are others in this file as well.